code/modules/mob/living/living.dm

```diff
@@ -1787,6 +1792,11 @@ GLOBAL_LIST_EMPTY(fire_appearances)
 		if(NAMEOF(src, body_position))
 			set_body_position(var_value)
 			. = TRUE
+		if(NAMEOF(src, current_size))
+			if(var_value == 0) //prevents divisions by zero.
```

Comment:
```suggestion
			if(var_value == 0) //prevents divisions of zero.
```

because it's not _by_ since var_value is the numerator. it's nasty anyways and should definitely not happen but that might cause some confusion on someone scavenging the code in three years from now

Reply:
Yea, but once the current size is set by 0 and then edited again, it becomes a division by zero. Anyway, good point.

Reply:
Yeah but if we block it in the first place it dont matter